components/stateTracker.js

```diff
@@ -90,6 +90,7 @@ StateTracker.prototype.bind = function() {
     var self = this;
     $(window).on('popstate', function() {
         if (!self.disabled) {
+            _.extend(history.state || {}, {_stateFromHistory: true});
```

Comment:
Popstate or Pushstate difference determines the normal user behavior. Why application can not depend of this?